source/docs/3/river_styling_expressions.rst

```diff
@@ -0,0 +1,109 @@
+Styling a river network with Expressions (QGIS3)
+===============================
+*Contributed by:* `Steven Kim <https://geographyclub.github.io/>`_
+
+The HydroRIVERS dataset provides seamless global coverage of consistently sized river reaches, supported by geometric information that allows for basic analysis of river network topology such as stream connectivity and distances. The data is a subset of the comprehensive HydroATLAS package from WWF.
```

Comment:
Move this description in the 'get the data' section where we download HydroRivers. The main description should reference the previous tutorial in the series and describe what we are doing in this tutorial. See this for reference and update https://www.qgistutorials.com/en/docs/3/block_world_styling.html